packages/eslint-config-react/lib/rules/react.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
         'react/jsx-handler-names': ['error', {
             eventHandlerPrefix: 'handle',
             eventHandlerPropPrefix: 'on',
+            checkLocalVariables: true,
```

Comment:
This setting is to cover `useCallback` handlers, right?

Reply:
Yep.